packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -299,13 +317,19 @@ export interface CommandletExecutor<T> {
 
 export abstract class SfdxCommandletExecutor<T>
   implements CommandletExecutor<T> {
+  protected showChannelOutput = true;
+
   protected attachExecution(
     execution: CommandExecution,
     cancellationTokenSource: vscode.CancellationTokenSource,
     cancellationToken: vscode.CancellationToken
   ) {
     channelService.streamCommandOutput(execution);
-    channelService.showChannelOutput();
+
+    if (this.showChannelOutput) {
```

Comment:
@Sweetman - Per our discussion the other day, this is one way we can hide the output channel if it is not necessary. I've made the default to show it always, which is how it is today so there will be no observable change of behavior.